tests/TestHarness/Cluster.py

```diff
@@ -265,10 +243,15 @@ def launch(self, pnodes=1, unstartedNodes=0, totalNodes=1, prodCount=1, topo="me
             tries = tries - 1
             time.sleep(2)
         loggingLevelDictString = json.dumps(self.loggingLevelDict, separators=(',', ':'))
-        cmd="%s %s -p %s -n %s -d %s -i %s -f %s --unstarted-nodes %s --logging-level %s --logging-level-map %s" % (
-            sys.executable, str(self.launcherPath), pnodes, totalNodes, delay, datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3],
-            producerFlag, unstartedNodes, self.loggingLevel, loggingLevelDictString)
+        cmd=(f'{sys.executable} {str(self.launcherPath)} -p {pnodes} -n {totalNodes} -d {delay} '
```

Comment:
@jgiszczak  With moving `launcher.py` you may need to check `self.launcherPath` in `Cluster.py` (line 117 above) as it is currently looking at `self.launcherPath = Path(__file__).resolve().parents[1] / "launcher.py"`

Reply:
Revised to fix generation of bridge topology files.

Reply:
2ff0243d3